src/assets/builders.json

```diff
@@ -23,5 +23,15 @@
     "name": "ng-deploy-azure",
     "description": "Deploy Angular apps to Azure using the Angular CLI",
     "repository": "https://github.com/Azure/ng-deploy-azure"
+  },
+  {
+    "name": "@angular/fire:deploy",
+    "description": "Deploy your application on Firebase Hosting<br><ul><li>add @angular/fire to your project</li><li>ng run [ANGULAR_PROJECT_NAME]:deploy to deploy</li></ul>",
```

Comment:
It's important to make sure that this specifies `ng add`, i.e.

> `ng add @angular/fire` to your project

Reply:
Done as part of #13